LICENSE

```diff
@@ -1,4 +1,7 @@
-See COPYRIGHT file for code ownership.
+BSD 3-Clause License
+
+Copyright (c) 2022, lsst
```

Comment:
Where did this come from? We want people to read the COPYRIGHT file. Also `lsst` is not a legal entity.

Reply:
lsst_versions

Reply:
Hilarious. That's a copy and paste error from some other package that I missed last time then.